### PR TITLE
Add commands for common links

### PR DESCRIPTION
**Pull request by namanyt**
this is base on issue #14

### CODE REVIEW

**Comment by namanyt** [comment]
Should implement commands for all links as mentioned by @IkBenOlie5:

- [x] docs
- [x] donate
- [x] facebook
- [x] github
- [x] odysee (formerly lbry)
- [x] patreon
- [x] plans
- [x] twitch
- [x] twitter
- [x] youtube

File: carberretta/bot/cogs/meta.py [hunk 1]
```diff
@@ -36,7 +36,26 @@ async def twitter_command(self, ctx: commands.bot.Context) -> None:
...
+    async def facebook_command(self, ctx: commands.bot.Context) -> None:
+        await ctx.send("https://facebook.carberra.xyz")
+        
+    @commands.command(name="lbry", aliases=["odysee"])
+    async def lbry_command(self, ctx: commands.bot.Context) -> None:
+        await ctx.send("https://lbry.carberra.xyz")
+    
+    @commands.command(name="donate")
+    async def donate_command(self, ctx: commands.bot.Context) -> None:
+        await ctx.send("https://donate.carberra.xyz")
```
Comment (parafoxia) [inline carberretta/bot/cogs/meta.py]: This should point to a new redirect I'm gonna set up for Streamlabs tips:
```suggestion
    @commands.command(name="donate", aliases=["tip"])
    async def donate_command(self, ctx: commands.bot.Context) -> None:
        await ctx.send("https://tip.carberra.xyz")
```

**Review by parafoxia — CHANGES_REQUESTED** [review]
Looks good, but just need to change one thing. Donation redirects are kinda weird so it's a little awkward lmao.

File: carberretta/bot/cogs/meta.py (hunk 1 above)
Comment (Jonxslays) [reply carberretta/bot/cogs/meta.py:54]: commands.Context, not commands.bot.Context.

Comment (namanyt) [reply carberretta/bot/cogs/meta.py]: is tip.carberra.xyz a working redirect site ? cus I cannot go to that link 😕

**Review by parafoxia — APPROVED** [review]

**Pull request merged**